bin/mac-docker-install.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+if [[ "$OSTYPE" != "darwin"* ]]
+then
+    echo "Your system isn't a Mac"
+    exit 1
+fi
+
+if ! type "docker-machine" > /dev/null
+then
+    echo "docker-machine command not found"
+    echo "try to download and install Docker4Mac"
+    echo "from: https://download.docker.com/mac/stable/Docker.dmg"
+    exit 1
+fi
+
+if type "dinghy" > /dev/null
+then
+    echo ""
+    echo "Dinghy lib detected!"
+    echo ""
+
+    read -p "Remove all trace of Dinghy Virtual machine? (y/n): " -n 1 -r
+    echo ""
+
+    if [[ $REPLY =~ ^[Yy]$ ]]
+    then
+        dinghy halt 2> /dev/null || true
+        dinghy destroy --force 2> /dev/null || true
+        echo " └─ Done"
+        echo ""
+    fi
+
+    read -p "Remove Dinghy lib? (y/n): " -n 1 -r
+    echo ""
+
+    if [[ $REPLY =~ ^[Yy]$ ]]
+    then
+        brew uninstall --force dinghy || true
+        echo " └─ Done"
+        echo ""
+    fi
+fi
+
+# Remove virtual machine with `default` name
+VIRTUAL_MACHINES=`docker-machine ls --quiet`
+
+if [[ ${VIRTUAL_MACHINES} =~ "default" ]]
+then
+    echo "Default virtual matched was found"
+    echo ""
```

Comment:
Instead of using`echo ""` instructions for a blank line, what do you think to add `\n` ?

Example : `echo "Default virtual matched was found\n"` (Maybe with the `-e` option)

Reply:
yes, you must add `-e` ;)

Reply:
done 👍